test/conformance/chainsaw/validate/clusterpolicy/standard/apicalls/default/policy.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: default
+spec:
+  rules:
+  - name: default-apicall
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+          operations:
+          - CREATE
+    context:
+    - name: testString
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{ request.namespace }}/invalid"
+        jmesPath: metadata.name
+        default: default
+    - name: testJSON
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{ request.namespace }}/invalid"
+        default: '{"metadata": {"name": "default"}}'
+    - name: testInteger
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{ request.namespace }}/invalid"
+        jmesPath: metadata.resourceVersion
+        default: 1     
+    validate:
+      deny:
+        conditions:
+          any:
```

Comment:
Can we change this to `all` to check all three conditions?

Reply:
I will change it to all!